From c323dd5087a15cb2b820fedf311ca288ec347878 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 8 Nov 2012 22:57:12 -0500
Subject: [PATCH] Center new windows

This makes more sense than letting mutter put them in the top
left corner.
---
 src/yelp-application.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/yelp-application.c b/src/yelp-application.c
index 2d004c2..e669661 100644
--- a/src/yelp-application.c
+++ b/src/yelp-application.c
@@ -444,6 +444,7 @@ application_uri_resolved (YelpUri             *uri,
 
         g_settings_get (settings, "geometry", "(ii)", &width, &height);
         window = yelp_window_new (data->app);
+        gtk_window_set_position (GTK_WINDOW (window), GTK_WIN_POS_CENTER);
         gtk_window_set_default_size (GTK_WINDOW (window), width, height);
         g_signal_connect (window, "resized", G_CALLBACK (window_resized), data->app);
         priv->windows = g_slist_prepend (priv->windows, window);
-- 
1.7.12.1


